docs/Developers/Git_Basics.rst

```diff
@@ -44,15 +44,25 @@ To do this, you can use these to clone the coala/coala-bears repositories:
 
 ::
 
-    $ git clone https://github.com/coala/coala
+    $ git clone -o upstream https://github.com/coala/coala
 
 or
 
 ::
 
-    $ git clone https://github.com/coala/coala-bears
+    $ git clone -o upstream https://github.com/coala/coala-bears
+
+.. note::
+
+    ``-o upstream`` sets the remote name of the original coala/coala-bears
```

Comment:
Yes, I guess you're right, it could be mistaken that way. You could submit an issue on that.